src/TLuaInterpreter.cpp

```diff
@@ -12248,6 +12249,15 @@ void TLuaInterpreter::initLuaGlobals()
     luaopen_yajl(pGlobalLua);
     lua_setglobal(pGlobalLua, "yajl");
 
+    // prepend profile path to package.path and package.cpath
+    QTimer::singleShot(0, [this]() {
+        luaL_dostring(pGlobalLua, QStringLiteral("package.path = getMudletHomeDir() .. [[%1?%1init.lua;]] .. package.path").arg(QDir::separator()).toUtf8().constData());
```

Comment:
Just do `auto separator = QDir::separator()).toUtf8().constData()` and then re-use that so it's cleaner.

Will test this soon!

Reply:
My bad, that's invalid code right there. Tired.